docs/sdks/client-sdks/javascript.md

```diff
@@ -70,16 +70,35 @@ If you are using the SDK for experiment assignments, make sure to pass in an ass
+| **`maxCacheAgeSeconds`** (number)                 | Maximum age, in seconds, previously cached configurations are considered valid and fetching an update configurations will be skipped                                                                                                                                                                                                   | `0`             |
+| **`useExpiredCache`** (boolean)                   | Consider initialization successfully complete without fetching updates, even if the configuration loaded from the cache is expired                                                                                                                                                                                                     | `false`         |
+| **`updateOnFetch`** (always\|expired\|empty)      | Sets how the configuration is updated after a successful fetch:<br/>• always - immediately start using the new configuration<br/>• expired - immediately start using the new configuration only if the current one has expired<br/>• empty - only use the new configuration if the current one is both expired and uninitialized/empty | `'always'`      |
+
+#### Configuration cache
+The SDK can cache previously loaded configurations for use in subsequent sessions, speeding up the time for the SDK to initialize.
+By default, a Local Storage-based cache is used in a browser environment and a Chrome Storage-based cache is used in a Chrome extension environment.
+A custom cache can be provided as the `persistentStore` initialization option.
+
+Caches have a concept of being "expired," dictated by the `maxCacheAgeSeconds` initialization option, and also overridable by the implementation of `isExpired()` for a custom persistent store.
+When the cached configuration is not expired, it is considered valid and no fetches will be made for updated configurations.
+
+When fetches are made--either from initialization or regular polling--you can control when the changes take effect with the `updateOnFetch` initialization option.
+It's default of `'always'` will start serving values from the most up-to-date configuration immediately. `'expired'` will only update if the cached configuration--now being used to serve assignments--has expired. 
+This is useful in combination with polling to prevent configurations in long-lived sessions from getting too stale. `'empty'` will only update if there is no pre-existing configuration (expired or not) and is 
+useful in combination with `useExpiredCache` for quickly initializing using a cached configuration and maintaining consistent assignments throughout the session even when the configuration is old. 
+It is essentially prefetching an updated configuration for the next session. 
```

Comment:
It would be useful to provide two example combinations of parameters: one matching what we want to do for eppo.cloud (fast init) and one if someone wants to have the most correct configuration always

Reply:
oh good idea!